controllers/imagerepository_controller.go

```diff
@@ -231,6 +231,51 @@ func (r *ImageRepositoryReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	return ctrl.Result{}, nil
 }
 
+func (r *ImageRepositoryReconciler) AddNotifications(ctx context.Context, imageRepository *imagerepositoryv1alpha1.ImageRepository) ([]imagerepositoryv1alpha1.NotificationStatus, error) {
+	log := ctrllog.FromContext(ctx).WithName("ConfigureNotifications")
+	log.Info("Configuring notifications", "Name", imageRepository.Name, "Namespace", imageRepository.Namespace)
```

Comment:
Should we print it after the check if any notifications configured?

Reply:
Fixed